dev/NavigationView/NavigationView.xaml

```diff
@@ -451,14 +453,22 @@
                                             <RowDefinition Height="Auto" />
                                             <RowDefinition Height="*" />
                                         </Grid.RowDefinitions>
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="Auto"/>
+                                            <ColumnDefinition Width="*"/>
+                                        </Grid.ColumnDefinitions>
 
                                         <Grid x:Name="ContentTopPadding"
-                                              Height="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.TopPadding}"
-                                              Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.LeftPaneVisibility}"/>
+                                            Grid.ColumnSpan="2"
+                                            Height="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.TopPadding}"
+                                            Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.LeftPaneVisibility}"/>
+
+                                        <Grid x:Name="ContentLeftPadding" Grid.Row="1"/>
```

Comment:
I prefer: Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.LeftXX}"

Reply:
@YuliKl, I don't think this is worth introducing a new TemplateSettings property, do you?

Reply:
@licanhua, I'm not sure whose Width you're referring to.

In general, template settings and theme resources serve different functions. The overall goal of the changes Regis is making are to allow app authors to change margins within NavigationView without digging into its control template. I don't think template settings make these types of modifications by apps easy.

Reply:
I don't know if your goal is width or margin, let me make width as an example.
For calculated value, the first choice is templatesettings.
If you just want customer to override, just do `<Grid x:Name="ContentLeftPadding" Grid.Row="1" Wdith={ThemeResource abc}`,
And we don't need even to specifiy x:Name for it.

Reply:
I know what you mean, Maybe introduce a new TemplateSettings property if a need arises in the future? 
We just want to set the Width of the new Grid element. There is no 'abc' resource that can be used. The width is computed on the fly based on the various buttons involved.

Reply:
Sorry, GitHub's auto-hiding of lines of code confused me. I'm now back on the right page :)

I don't actually have an opinion in this case because I don't really expect apps to want to change ContectLeftPadding's width. Sounds like Regis is saying that we don't have a value to save into template settings - so let's not. But this feels like an implementation detail and I trust you, the developers, to do the right thing.